clash-starters/clash-example-project/src/Example/Project.hs

```diff
@@ -0,0 +1,9 @@
+module Example.Project where
+
+import Clash.Prelude
+
+plus :: Signed 8 -> Signed 8 -> Signed 8
+plus a b = a + b
+
+topEntity :: Signed 8 -> Signed 8 -> Signed 8
```

Comment:
I think it would be nice to see a comment over `topEntity`, something that points out that this is the clash equivalent of `main`, it can have any (monomorphic) type etc.